pp_hot.c

```diff
@@ -1077,16 +1077,15 @@ PP(pp_multiconcat)
 
                 /* if both args are the same magical value, make one a copy */
                 if (left == right && SvGMAGICAL(left)) {
-                    left = sv_newmortal();
+                    SV * targetsv = right;
                     /* Print the uninitialized warning now, so it includes the
                      * variable name. */
                     if (!SvOK(right)) {
                         if (ckWARN(WARN_UNINITIALIZED))
                             report_uninit(right);
-                        sv_setbool(left, FALSE);
+                        targetsv = boolSV(FALSE);
```

Comment:
While it would make no difference to the compiled code, would the following not be conceptually simpler?
```c
targetsv = &PL_sv_no;
```

Reply:
Yes, that makes sense to me.

Reply:
Thanks for the suggestion. Changed, rebased, and pushed.